.jenkins/pytorch/common.sh

```diff
@@ -19,6 +19,7 @@ if [[ "${BUILD_ENVIRONMENT}" == *rocm* ]] && [[ "${BUILD_ENVIRONMENT}" =~ py((2|
   shopt -s expand_aliases
   export PYTORCH_TEST_WITH_ROCM=1
   alias python="$PYTHON"
+  python -m pip install hypothesis # hypothesis was not found in jenkins pytorch-linux-bionic-rocm3.8-py3.6-test1
```

Comment:
Before adding this, the pr/pytorch-linux-bionic-rocm3.8-py3.6 job was failing as the tests could not find the hypothesis package (even though it was installed in Docker). Adding this line fixes it, though there may be a more elegant solution.

Reply:
Though the ROCm CI image will be rebuilt successfully by this PR, our ROCm CI is still Jenkins-based and the docker image hash is maintained in the ossci-job-dsl repo.  See https://github.com/pytorch/ossci-job-dsl/pull/78 for an example diff where we update the CI image and hash.

Reply:
Thank you--I've opened a PR [here](https://github.com/pytorch/ossci-job-dsl/pull/80)